From b4c1bdbd775d49fc21738485bb8ba0ffb5400060 Mon Sep 17 00:00:00 2001
From: rackhousetavern <shockt88+rackhousetavern@gmail.com>
Date: Sun, 17 Mar 2013 13:44:04 -0500
Subject: [PATCH 5/7] Update CNAME

---
 CNAME | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CNAME b/CNAME
index 268f35f..a1436f8 100644
--- a/CNAME
+++ b/CNAME
@@ -1 +1 @@
-rackhousetavern.com
+_rackhousetavern.com
-- 
1.8.1.4

